app/src/main/res/values/dimens.xml

```diff
@@ -1,7 +1,9 @@
 <resources>
-    <!-- Default screen margins, per the Android Design guidelines. -->
+    <!-- Spacings -->
     <dimen name="activity_horizontal_margin">16dp</dimen>
     <dimen name="activity_vertical_margin">16dp</dimen>
-    <dimen name="size_text_activities">25sp</dimen>
-    <dimen name="padding">16px</dimen>
+    <dimen name="padding">16sp</dimen>
```

Comment:
The word `padding` is way too generic for a dimen. Also, the `sp` unit should only be used for fonts, as it is related to the devices' font size. For example, if the user goes to the the device's settings and changes the font size to be huge, then any view that is using this dimen based on `sp` will become larger as well.

For most dimens, the unit `dp` should be used. DP stands for Density Pixels and is related to the devices's physical screen resolution. This provides a more consistent look between devices and can't be changed by the user. You can read more about this in Android's docs online.

We should have a set of reusable dimens like this:

```xml
<!-- Spacings -->

<dimen name="spacing_tiny">2dp</dimen>
<dimen name="spacing_small">4dp</dimen>
<dimen name="spacing_medium">8dp</dimen>
<dimen name="spacing_large">16dp</dimen>
<dimen name="spacing_extra_large">32dp</dimen>
<dimen name="spacing_huge">72dp</dimen>

<!-- Fonts -->

<dimen name="font_small">14sp</dimen>
<dimen name="font_medium">16sp</dimen>
<dimen name="font_large">20p</dimen>
<dimen name="font_extra_large">24sp</dimen>
<dimen name="font_huge">32sp</dimen>
```

With a set of dimens like this, we can create most of the UIs without having to hardcode dimens anywhere.